include/arch/arm/cortex_m/mpu/arm_core_mpu_dev.h

```diff
@@ -34,6 +34,8 @@ extern "C" {
 enum {
 #ifdef CONFIG_NOCACHE_MEMORY
 	NOCACHE_MEMORY_REGION,
+#ifdef CONFIG_RAM_FUNCTION
+	RAM_FUNCTION_REGION,
 #endif
```

Comment:
It also looks like there is a missing #endif here.

Reply:
Fixed.